rpa-recorder/6.1/modules/ROOT/pages/user-interface.adoc

```diff
@@ -114,6 +114,37 @@ To continue recording, add a Bot Task.
 [NOTE]
 Pause the recording before adding an activity or changing the creation mode of the activities. If you open the pop-up using the respective hotkey, while the recording is running, the recording automatically pauses.
 
+== Web Recording Browser
+
+Use the Web Recording Browser to record web elements. It behaves like a normal browser and enables you to select additional elements of a website and determine their XPath.
+You can also save your attribute selection of a web element to reuse it when recording the same web elements on a website.
+
+Adding a web task opens the Web Recording Browser. If you choose to continue using the session of the last closed Web Task, the browser loads the URL that is defined as the initial URL for the enclosing Web Session.
```

Comment:
This is not how it works. This sounds to me a bit like RPA Builder documentation. There is no such thing as initial URL or enclosing Web Session in RPA Recorder.
What happens is then when you add a new Web Task AND there was already a web task before that, then the checkbox "Continue with already existing web session." is active (otherwise, for the first web task it's disabled). If user checks this checkbox, the last loaded browser page from the previous web task gets displayed again. The browser does not load any URL in this case, it only let you continue where you've stopped in the previous web task.
(Compare this with opening a new blank tab in your Web Browser vs. continuing with an existing tab with some page which is already open there)